core/src/main/scala/pureconfig/BasicReaders.scala

```diff
@@ -95,7 +96,7 @@ trait RegexReaders {
 
 /** Trait containing `ConfigReader` instances for `java.time` classes.
   */
-trait JavaTimeReaders {
+trait JavaTimeReaders extends JavaEnumReader {
```

Comment:
An alternative which avoids having `JavaTimeReaders` extend `JavaEnumReader` is to make a companion object for `JavaEnumReader` , i.e.

```scala
object JavaEnumReader extends JavaEnumReader`
```

at which point you can just `import JavaEnumReader._`. In general it can be argued that these traits should have companion objects since they don't require any parameters but I think thats a separate discussion.

Reply:
If you follow my suggestion above of moving `baseJavaEnumReader` to `pureconfig.configurable` you won't have this problem :)